lib/libutee/assert.c

```diff
@@ -41,7 +41,7 @@ void _assert_log(const char *expr __maybe_unused,
 				expr, file, line, func);
 }
 
-void __noreturn _assert_break(void)
+void _assert_break(void)
```

Comment:
OK, then we know how to make sparse happy about `_assert_break()`.

But there's one problem left, the __noreturn attribute is contagious so if `assert()` is used with an expression that the compiler knows always is false it will cause a warning unless the function doesn't have the `__noreturn` attribute too. Then we have `NDEBUG=1` where `assert()` is a nop and the function has wrong attribute again.
This is annoying and I think we should do something about it, but we can take that later if you prefer that.